x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -766,15 +781,53 @@ class TimeseriesChartIntl extends Component {
         )
       );
 
+    const that = this;
+
     // Remove dots that are no longer needed i.e. if number of chart points has decreased.
     dots.exit().remove();
     // Create any new dots that are needed i.e. if number of chart points has increased.
     dots
       .enter()
       .append('circle')
       .attr('r', LINE_CHART_ANOMALY_RADIUS)
+      .on('click', function (d) {
```

Comment:
I'm getting an error if I try to use the 'Configure job rules' menu item.

```
react-dom.development.js:4091 Uncaught TypeError: props.showRuleEditorFlyout is not a function
```

Reply:
Fixed the rule editor flyout in [5be389b](https://github.com/elastic/kibana/pull/175556/commits/5be389b92e14d66c8c4633449937a3ae111d499a).